From 693f3b862bfbf3193a2e698488afb2970dfff677 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Fri, 31 May 2019 07:38:29 +0200
Subject: [PATCH 1/1] efi_loader: export efi_set_time()

To let a board implement the runtime version of SetTime() we have to
provide the definition of the weak function in an include.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 include/efi_loader.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/efi_loader.h b/include/efi_loader.h
index 77b2f60bdc..23ce732267 100644
--- a/include/efi_loader.h
+++ b/include/efi_loader.h
@@ -594,6 +594,8 @@ efi_status_t __efi_runtime EFIAPI efi_get_time(
 			struct efi_time *time,
 			struct efi_time_cap *capabilities);
 
+efi_status_t __efi_runtime EFIAPI efi_set_time(struct efi_time *time);
+
 #ifdef CONFIG_CMD_BOOTEFI_SELFTEST
 /*
  * Entry point for the tests of the EFI API.
-- 
2.20.1

